<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_data_to_loe</class>
 <widget class="QWidget" name="add_data_to_loe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>Введите данные, которое будут добавлены</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>ФИО</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="employee_FIO"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Должность</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="employee_post"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Оклад</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="employee_salary"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Отдел</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="employee_departments"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>15</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Добавить данные</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
